### PR TITLE
(another) site

### PR DESCRIPTION
**Pull request by quitd**

### CODE REVIEW

**Comment by RawrbearHS** [comment]
Thank you so much! This looks great!

I'd like to propose a couple of changes, if possible:

- The hamburger menu should be a bit bigger, and easy to press.
- The header that says "Balloons" and the hamburger menu should be vertically centered in the nav.
- Could you make the header font just a little bit bigger? That way, it's bolder and easier to read.
- Could you make the hamburger menu background color white enough? - it clashes with the on-page content without it.

image

If it's too much of a bother, I could try to look into it, but this is awesome so far. Thank you for your work on this!

**Comment by quitd** [comment]
can you review 1dbc154?

File: style.css [hunk 1]
```diff
@@ -33,14 +33,15 @@ body {
...
-  font-size: .7em;
+  font-size: .8em;
   position: fixed;
   padding: 1em;
   top: 0;
   left: 0;
   width: 100%;
   background: linear-gradient(to right, hsla(0, 0%, 100%, 0.32), hsla(0, 0%, 88%, 0.5));
   backdrop-filter: blur(5px);
+  vertical-align: middle;
```
Comment (RawrbearHS) [inline style.css:44]: Looks like this is aligned, but not the logo text.

Comment (RawrbearHS) [reply style.css:44]: I feel like the logo text could also be a little bigger to match.

**Review by RawrbearHS — APPROVED** [review]
Approving for now - will help with polish later.

**Comment by RawrbearHS** [comment]
I'm gonna figure out how to put this in a gh-pages branch lol. (Reverting the revert...)

**Pull request merged**